docs_src/microservices/core/metadata/Ch-Metadata.md

```diff
@@ -4,37 +4,33 @@
 
 Specifically, metadata has the following abilities:
 
--   Manages information about the devices connected to, and
-    operated by, EdgeX Foundry
+-   Manages information about the devices connected to, and operated by, EdgeX Foundry
 -   Knows the type, and organization of data reported by the devices
 -   Knows how to command the devices
 
 Although metadata has the knowledge, it does not do the following activities:
 
--   It is not responsible for actual data collection from
-    devices, which is performed by device services and core
-    data
--   It is not responsible for issuing commands to the
-    devices, which is performed by core command and device
+-   It is not responsible for actual data collection from devices, which is performed by device services and core data
+-   It is not responsible for issuing commands to the devices, which is performed by core command and device
     services
 
 ## Data Models
 
 To understand metadata, its important to understand the EdgeX data objects it manages.  Metadata stores its knowledge in a local persistence database.  [Redis](https://redis.io/) is used by default, but a database abstraction layer allows for other databases to be used.
 
-!!! Note
-        EdgeX has used [MongoDB](https://www.mongodb.com/) in the past. Through the Geneva release, MongoDB is still supported but is considered deprecated.  Alternate (both open source and commercial) implementations have also been provided in the past. 
-
 ### Device Profile
 
 Device profiles define general characteristics about devices, the data they provide, and how to command them. Think of a device profile as a template of a type or classification of device. For example, a device profile for BACnet thermostats provides general characteristics for the types of data a BACnet thermostat sends, such as current temperature and humidity level. It also defines which types of commands or actions EdgeX can send to the BACnet thermostat.  Examples might include actions that set the cooling or heating point.  Device profiles are typically specified in YAML file and uploaded to EdgeX.  More details are provided below.
 
+!!! edgey "EdgeX 2.0"
+    The device profile was greatly simplified in EdgeX 2.0 (Ireland).  There are now just two sections of the document (deviceResources and deviceCommands) versus the three (deviceResources, deviceCommands and coreCommands) of EdgeX 1.x profiles.  Device resources and device commands are made available through the core command service with the `isHidden` property on either is set to fault.  This makes a core command section no longer necessary in EdgeX 2.
+    
+    However, this does mean that EdgeX 2 profiles are not backward compatible and EdgeX 1.x profiles must be migrated.  See the migration guides for details.
```

Comment:
Add link to the migration guide?

Reply:
added a "placeholder" suggesting migration guide coming soon.